packages/ui/src/components/primitives/base/Icons.tsx

```diff
@@ -139,15 +139,15 @@ export const ExternalLinkIcon = () => {
 	);
 };
 
-export const CopyIcon = () => {
+export const CopyIcon = ({ className }: { className?: string }) => {
```

Comment:
Headless icons probably has a copy icon we can use instead of duplicating it here

Reply:
:+1: